src/runtime_src/xocl/CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ include_directories(
   ${XRT_XOCL_API_DIR}
 )
 
+add_compile_options("-Wno-nonnull")
```

Comment:
What's the reason for disabling this warning?

Reply:
The code is in boot devel package, not in XRT.

Reply:
I don't understand what that means?  This is a CMakeList.txt in XRT.

Reply:
I take it back. I was looking at a different log. You are right. I see the issue here:
 499   | if (flags & CL_MEM_COPY_HOST_PTR)
  500   | | std::memcpy(m_host_ptr,host_ptr,sz);
There is no null check. I will fix this code and then remove this supression. Will push a new commit.